images/code-build-image/dockerd-entrypoint.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
```

Comment:
Do you want to include this file even though you are not using in your implementation?

Reply:
Yes...for now a reference